sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1107,6 +1114,43 @@ Exceptions:
 
 |===
 
+==== New Event Member Functions
+
+Table {counter: tableNumber}. Additional member functions of the `sycl::event` class.
+[cols="2a,a"]
+|===
+|Member function|Description
+
+|
+[source,c++]
+----
+template <typename Param>
+typename Param::return_type event::ext_oneapi_get_profiling_info(ext::oneapi::experimental::node Node) const;
```

Comment:
We should add some line breaks to this declaration, in the rendered view it makes this column so long that the description in the next column is hard to read.

Reply:
I've changed `ext::oneapi::experimental::node` to `node` as is already the case for other functions.